index.src.html

```diff
@@ -512,52 +565,12 @@
           playing</a>.
         </p>
         <p>
-          The algorithm to <dfn>convert the depth map value to grayscale</dfn>,
-          given a depth map value <var>d</var>, is as follows:
-        </p>
-        <ol>
-          <li>Let <var>near</var> be the the <a>near value</a>.
-          </li>
-          <li>Let <var>far</var> be the the <a>far value</a>.
-          </li>
-          <li>Apply the <a>rules to convert using range inverse</a> to
-          <var>d</var> to obtain quantized value <var>d<sub>8bit</sub>.</var>
-          </li>
-          <li>Return <var>d<sub>8bit</sub></var>.
-          </li>
-        </ol>
-        <p>
-          The <dfn>rules to convert using range inverse</dfn> are as given in
-          the following formula:
-        </p>
-        <p>
-          `d_(n) = (far * (d - n ear)) / (d * (far - n ear))`
+          The <code>video</code> element draws the depth map at the red channel
```

Comment:
If we use the red channel, we should probably mention:

`<div class="note">It is an implementation detail how the frames are rendered to the screen. For example, the implementation may use a grayscale representation.</div>`